### PR TITLE
CRUD for datacenter_rack_role and datacenter_rack

### PR DESCRIPTION
**Pull request by sungo**

### CODE REVIEW

File: Conch/lib/Conch/Controller/DatacenterRack.pm [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+}
+
+=head2 get
+
+Get a single rack
+
+=cut
+
+sub get ($c) {
```
Comment (lseppala) [inline Conch/lib/Conch/Controller/DatacenterRack.pm]: For both get and get_all, should restrict to global admin.

You can also pull this logic into `under`, which allows you to de-dup the check for update and delete

File: Conch/lib/Conch/Controller/DatacenterRackRole.pm [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+}
+
+=head2 get
+
+Get a single rack role
+
+=cut
+
+sub get ($c) {
```
Comment (lseppala) [inline Conch/lib/Conch/Controller/DatacenterRackRole.pm]: Likewise, should restrict to global admin I think.

File: Conch/lib/Conch/Controller/DatacenterRack.pm (hunk 1 above)
Comment (sungo) [reply Conch/lib/Conch/Controller/DatacenterRack.pm]: Fixed.

File: Conch/lib/Conch/Controller/DatacenterRackRole.pm (hunk 2 above)
Comment (sungo) [reply Conch/lib/Conch/Controller/DatacenterRackRole.pm]: Fixed.

File: Conch/lib/Conch/Controller/DatacenterRack.pm [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+	$c->status(200, Conch::Model::DatacenterRack->all());
+}
+
+
+=head2 update
+
+=cut
+
+sub update ($c) {
+	return $c->status(403) unless $c->is_global_admin;
```
Comment (lseppala) [inline Conch/lib/Conch/Controller/DatacenterRack.pm]: Dup from `under` logic. Same with `delete`

**Pull request merged**